ui/src/components/Header/Controller.tsx

```diff
@@ -32,49 +38,110 @@ export const Controller = (): ReactElement => {
     }
   }, [isLoading]);
 
-  const start = async () => {
-    const images = await ddClient.docker.listImages() as [DockerImage];
-    if (!images.some(image => image.RepoTags?.at(0) === 'localstack/localstack:latest')) {
-      ddClient.desktopUI.toast.warning('localstack/localstack:latest not found; now pulling..');
-    } else {
-      ddClient.desktopUI.toast.success('Starting LocalStack');
-    }
-    const corsArg = '-e EXTRA_CORS_ALLOWED_ORIGINS=http://localhost:3000';
+  const normalizeArguments = async () => {
+    const extendedFlag = FLAGS;
+
+    const corsArg = ['-e', `EXTRA_CORS_ALLOWED_ORIGINS=${CORS_ALLOW_DEFAULT}`];
     const addedArgs = runConfig.find(config => config.name === runningConfig)
```

Comment:
could we break this section up into multiple lines doing separate things? 
feels a bit convoluted...